hydra-node/src/Hydra/HeadLogic/Outcome.hs

```diff
@@ -126,7 +126,7 @@ data StateChanged tx
   | DecommitFinalized
       { chainState :: ChainStateType tx
       , headId :: HeadId
-      , decommitTxId :: TxIdType tx
+      , distributedOutputs :: [TxOutType tx]
```

Comment:
Why is this not `UTxOType tx`? It would be consistent with fanout

Reply:
To do that we would need to know how to construct the correct _TxIn_ part of the `UTxO` if we want to make assertions in our tests about it. Since the increment tx is posted by the hydra-node _automatically_ we don't have enough information locally to make those tests assertions but could match on other fields. Do you see this as important or the list of `TxOut` is fine? Other observations are also all over the place and the information in there is not uniform.

Reply:
Scratch my last message, I wanted to implement the decrement observation using `UTxO` and ended up rewriting couple of related tests.